linked_list/linked_list.cpp

```diff
@@ -0,0 +1,426 @@
+/*
+
+* C++ Program to Implement Linked List
+
+*/
+
+#include<iostream>
+#include<cstdio>
+#include<cstdlib>
+
+using namespace std;
+
+/*
+
+* Node Declaration
+
+*/
+struct node
+{
```

Comment:
Coding style should not differ in a single repository. See other C++ codes and make changes accordingly.
For ref. see Trie.cpp
https://github.com/iiitv/algos/blob/master/trie/trie.cpp